sllurp/llrp_proto.py

```diff
@@ -1965,6 +2060,29 @@ def encode_TagObservationTrigger(par, param_info):
                                       timeout)
 
 
+def decode_TagObservationTrigger(data, name=None):
+    logger.debugfast("decode_TagObservationTrigger")
+
+    (
+        trigger_type,
+        _,
+        number_of_tags,
+        number_of_attempts,
+        t,
+        timeout
+    ) = tagobservationtrigger_unpack(data)
+
+    par = {
+        'TriggerType': trigger_type,
```

Comment:
Here you could use:
TagObservationTrigger_Type2Name
to transform value to name.
This dict does not exist yet, but you can easily create it by reversing:
TagObservationTrigger_Name2Type
(There are example with other dicts around this one)